lib/CL/pocl_llvm_api.cc

```diff
@@ -854,10 +1022,18 @@ int pocl_llvm_get_kernel_metadata(cl_program program,
                               content.str().size());
 #endif
 
+#if defined(LLVM_OLDER_THAN_3_9)
```

Comment:
Both functions are needed in llvm3.9, because spir examples still use the "opencl.kernels" meta.